src/v2/Apps/Example/Routes/ExampleArtist/ExampleArtistApp.tsx

```diff
@@ -15,18 +18,49 @@ const ExampleArtistApp: React.FC<ExampleArtistAppProps> = ({ artist }) => {
       <Text variant="title" mb={2}>
         {artist.name}
       </Text>
+      <FollowArtistButton
+        artist={artist}
+        contextModule={ContextModule.artistHeader}
+      />
       <Text>{artist.bio}</Text>
     </Box>
   )
 }
 
+/**
+ * Routes with /:id require an additional AnalyticsContext.Provider
+ * declaration to add slug and id, extending the context provided by <Boot>
```

Comment:
This is good to know, i wasn't aware 👍

Reply:
Had tried to get this out of the routes, but is a bit trickier than i had hoped - not the ideal solution but it works